src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonDateTimeTestData.cs

```diff
@@ -215,6 +214,7 @@ public static IEnumerable<object[]> InvalidISO8601Tests()
             yield return new object[] { "\"1997-07-16T19:20:30.45555555550000000\"" };
             yield return new object[] { "\"1997-07-16T19:20:30.45555555555555555\"" };
             yield return new object[] { "\"1997-07-16T19:20:30.45555555555555555555\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555555555555+01:300\"" };
```

Comment:
I don't think we have `DateTimeOffset` failure tests for leading whitespace. We'd take a follow up to add these.